src/components/BarChartCard/_bar-chart-card.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../globals/vars';
+
+.#{$iot-prefix}--bar-chart-container {
+  padding-left: 16px;
```

Comment:
we need to use the carbon spacing tokens here. This i believe is `$spacing-05`. WE should consolidate all these padding styles as well. `padding: 0 $spacing-05 $spacing-05

Reply:
thanks, fixed!